frontend/src/pages/dashboard/Dashboard.tsx

```diff
@@ -1,15 +1,16 @@
 import { DashboardFilters } from './DashboardFilters';
 import { useTable } from '@/hooks';
-import { useRecommitmentCycle, useRoleContext } from '@/providers';
+import { useRoleContext } from '@/providers';
 import type { Status } from '@/common';
 import { Role } from '@/common';
 import { useFilters } from '@/hooks/useFilters';
 import { Table, TableFooterNav, TableFooterPageSelect, Tabs } from '@/components';
 
 import { RecommitmentDashBanner } from '@/components/profile/banners/RecommitmentDashBanner';
+import { useRecommitmentCycle } from '@/hooks/useRecommitment';
```

Comment:
only called on the dashboard and profile pages, rather than immediately on login